appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -191,7 +191,8 @@ internal class TooltipTrait(
         path: Path,
         isDark: Boolean
     ) = this.then(
-        if (style?.borderWidth != null && style.borderWidth ne 0.0 && style.borderColor != null) {
+        @Suppress("ComplexCondition")
+        if (style?.borderWidth != null && style.borderWidth ne 0.0 && style.borderColor != null && !path.isEmpty) {
```

Comment:
I considered other ways to rework this function but all of them led to what felt like an overall less readable solution than just the pretty simple 4 case conditional check here, so, I suppressed the Detekt warning (which wants 3 conditions max)

Reply:
interesting, the isEmpty might be just what we need here to be real, what happens is that the first pass ever the containerDimens is empty. I did not find any way to go around this yet. we wait until the "content" returns a size to update containerDimens.value, which re-composes all these modifiers.